bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -483,8 +483,11 @@ def from_api_repr(cls, resource):
         :returns: instance
         """
         name = resource.get('name')
-        array_type = resource['parameterType']['arrayType']
-        values = resource['parameterValue']['arrayValues']
+        array_type = resource['parameterType']['arrayType']['type']
```

Comment:
No better way to do it, I just worry about missing keys and opaque error messages to users:

```python
>>> d = {}
>>> d['a']['b']['c']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'a'
>>> d['a'] = da = {}
>>> d['a']['b']['c']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'b'
>>> da['b'] = db = {}
>>> d['a']['b']['c']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'c'
>>> db['c'] = 'sheesh'
>>> d['a']['b']['c']
'sheesh'
```